L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -151,6 +149,8 @@
 
 # HYBRID: prompt tracking
 if (L1TRKALGO == 'HYBRID'):
+    process.TrackTriggerSetup.GeometricProcessor.ChosenRofZ = 50.0
```

Comment:
Why is the Hybrid depending on the TMTT GP? And why is the eta range of the Hybrid being set to 2.4? I seem tp recall the L1TrkNtuple_eff_eta.pdf plot showing non-zero efficiency up to eta 2.5 in the past, although that currently doesn't seem to be the case. Is the eta cut in TrackTriggerSetup used only in the DTC & KF? It's also dodgy changing the default value of parameters for the baseline algo in a cfg.py, as anyone else running this algo will have to copy these parameter mods.

Reply:
performing those changes (restoring the past configuration) is dodgy to begin with, they are just made to pass CI.

Reply:
Please make the CI more reluctant so that we don't need these lines of code.